README.md

```diff
@@ -137,10 +137,10 @@ This project is made possible thanks to the wonderful work by the [mozilla/uniff
 [`bdk`]: https://github.com/bitcoindevkit/bdk
 [`bdk-ffi`]: https://github.com/bitcoindevkit/bdk-ffi
 ["Getting Started (Developer)"]: https://github.com/bitcoindevkit/bdk-ffi#getting-started-developer
-[bdk-jvm]: https://search.maven.org/artifact/org.bitcoindevkit/bdk-jvm/0.11.0/jar
-[bdk-android]: https://search.maven.org/artifact/org.bitcoindevkit/bdk-android/0.11.0/aar
-[bdk-jvm (Maven Central)]: https://central.sonatype.dev/artifact/org.bitcoindevkit/bdk-jvm/0.11.0
-[bdk-android (Maven Central)]: https://central.sonatype.dev/artifact/org.bitcoindevkit/bdk-android/0.11.0
+[bdk-jvm]: https://search.maven.org/artifact/org.bitcoindevkit/bdk-jvm/1.1.0/jar
+[bdk-android]: https://search.maven.org/artifact/org.bitcoindevkit/bdk-android/1.1.0/aar
+[bdk-jvm (Maven Central)]: https://central.sonatype.com/artifact/org.bitcoindevkit/bdk-jvm/1.1.0
+[bdk-android (Maven Central)]: https://central.sonatype.com/artifact/org.bitcoindevkit/bdk-android/1.1.0
```

Comment:
I notice that the first link (`search.maven.org...`) now redirects to the second one. You are also correct that we can remove the version from the URL and the link will automatically point to the latest. I suggest the following lines instead:

```md
[bdk-jvm]: https://central.sonatype.com/artifact/org.bitcoindevkit/bdk-jvm/
[bdk-android]: https://central.sonatype.com/artifact/org.bitcoindevkit/bdk-android/
```

And the table that uses the old link above be updated to use those instead.

Reply:
i.e. lines 24 and 25 be fixed to use those links instead.

Reply:
not sure if I'm reading your comments correctly, but here's what I changed it to:

```
[bdk-jvm]: https://central.sonatype.com/artifact/org.bitcoindevkit/bdk-jvm/
[bdk-android]: https://central.sonatype.com/artifact/org.bitcoindevkit/bdk-android/
[bdk-jvm (Maven Central)]: https://central.sonatype.com/artifact/org.bitcoindevkit/bdk-jvm/
[bdk-android (Maven Central)]: https://central.sonatype.com/artifact/org.bitcoindevkit/bdk-android/
```